### ISSUE TITLE
Internal frame??

### ISSUE THREAD

**Issue by Nugss** [hrydgard/ppsspp#15546]
### Game or games this happens in

ULUS10213 - The warriors / ULUS10160 GTA VCS

### What area of the game / PPSSPP

I don't know what to call it but, on the latest build the internal frame suddenly drops example 5/5, 16/16, 20/20. The game plays like with frameskipping without auto frameskip.

I just tried it on the 2 games I have same situation happens, so possibly it happens on other games too

### What should happen

I can't remember what is the last build I downloaded before but it runs the 2 games smooth even on low end device

### Logs

![Screenshot_2022-05-20-11-59-38-88](https://user-images.githubusercontent.com/97185681/169450640-945154d9-3a0d-4f7f-8b2d-0448648d0f85.png)
![Screenshot_2022-05-20-11-59-45-83](https://user-images.githubusercontent.com/97185681/169450695-e11df5a4-d5cb-415b-b0bc-617d46019731.png)

### Platform

Android

### Mobile phone model or graphics card

Oppo a3s

### PPSSPP version affected

v1.12.3-1190-g4c6f6b63d

### Graphics backend (3D API)

Vulkan

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [ ] Try resetting settings or older versions and include if the issue is related.
- [X] Try without any cheats and without loading any save states.
- [X] Include logs or screenshots of issue.

**Comment by ANR2ME** [comment]
Please try to find out out the last working version.
May be started from stable release 1.12.3 -> 1.11.3 -> 1.10.3

**Comment by hrydgard** [comment]
Would be very useful if you could find approximately in what build it started happening for you, and if you could experiment a bit with settings to see if there's a particular one that causes it..

**Comment by Nugss** [comment]
I can't remember the version I last downloaded but the builds from april will work properly

**Comment by Nugss** [comment]
The build v1.12.3-1109-g815910c6f from date 4/30/22 works well same like the version I use before

**Comment by ANR2ME** [comment]
Btw did you have Auto-frameskipping enabled? 
if it's enabled try disabling it (since you said your device can play the game smoothly without frameskipping)

**Comment by unknownbrackets** [comment]
Do you have "Force real clock sync (slower, less lag)" enabled or any locked CPU speed?  In general it'd help to know what settings you're using.

-[Unknown]

**Comment by Nugss** [comment]
> 

Yes my device can play games smooth but it lags on some angle so I need frameskipping and auto frameskip enabled to run game with 100% speed at least.

Sure I'll try that version later

**Comment by Nugss** [comment]
> Do you have "Force real clock sync (slower, less lag)" enabled or any locked CPU speed? In general it'd help to know what settings you're using.
> 
> -[Unknown]

I'll tried to enable that on latest build and it become worse, 100% speed but the internal frame/frames is become lower, just like 5 frameskipping without auto frameskip :(

**Comment by Gamemulatorer** [comment]
This might be related/duplicate of https://github.com/hrydgard/ppsspp/issues/15525

**Comment by ANR2ME** [comment]
> > 
> 
> Yes my device can play games smooth but it lags on some angle so I need frameskipping and auto frameskip enabled to run game with 100% speed at least.
> 
> Sure I'll try that version later

On GTA VCS without Frameskipping at all, it's normal for the FPS to go down for a short time when making a turn where there are a lot of buildings in the view, and when Auto-frameskipping is enabled (on the latest dev build) this will make the internal FPS to go down too to keep up 100% performance (which is why i'm seeing 25/25 sometimes instead of 30/30), so this is a normal/correct behavior of Auto-frameskipping i think.

**Comment by Nugss** [comment]
Yeah but on latest build has more internal frame drops on low, i don't know how, but it suddenly happens even inside of safe house same on the warriors, even on the place or angle that usually smooth it happens the same way too. Since I'm low end device i need the frameskipping and auto frameskip on too run the game in 100%. But on that latest build when I turn that on the frame suddenly drop down that is not usual.
